lib/study.js

```diff
@@ -0,0 +1,54 @@
+const self = require('sdk/self');
+const shield = require('shield-studies-addon-utils');
+const { when: unload } = require('sdk/system/unload');
+
+const { Advisor } = require('./Advisor');
+
+let Advise = null;
+
+const studyConfig = {
+  name: self.addonId,
+  duration: 7,
+  surveyUrls: {
+    'end-of-study': 'mozilla.org',
+    'user-ended-study': 'mozilla.org',
+    ineligible: null,
+  },
+  variations: {
+    'passive-panel'() {
+      Advise.start('passivePanel');
+    },
+    'auto-open-panel'() {
+      Advise.start('autoOpenPanel');
+    },
+    'observe-only'() {},
```

Comment:
This is now [created by default in the `Study` class constructor](https://github.com/mozilla/shield-studies-addon-utils/blob/7b938ace2897fba993d814fe143e1306cfc287b5/lib/index.js#L143-L150), so isn't technically needed. @gregglind - do we want to set a precedence of study add-ons explicitly declaring the `observe-only` variation, or leave it out of the studies for the sake of brevity?